tests/utils.go

```diff
@@ -2050,11 +2052,17 @@ func AddEphemeralCdrom(vmi *v1.VirtualMachineInstance, name string, bus string,
 }
 
 func NewRandomFedora32VMIWithFedoraUser() *v1.VirtualMachineInstance {
-	vmi := NewRandomVMIWithEphemeralDiskAndUserdata(ContainerDiskFor(ContainerDiskFedora), `#!/bin/bash
-	    echo "fedora" |passwd fedora --stdin
-        echo `)
-	vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512M")
-	return vmi
+	userData := `#!/bin/bash
+		echo "fedora" |passwd fedora --stdin
```

Comment:
Remove tabs there they are inserted at the userData

Reply:
Done